### ISSUE TITLE
App crashes every time I pull down quick settings

### ISSUE THREAD

**Issue by Efreak** [emrsmsrli/Caffeine#7]
Apk source: F-Droid
Device: Moto G6 (stock/unrooted)
Android 8.0.0
Build number: OPS27.104-92

Relevant logcat obtained from `adb logcat|grep -E '6716|caffeine'`. Can send full logcat if you send me an email (or keybase chat)0
```
12-04 19:10:37.006  1719  2862 I ActivityManager: Start proc 6716:tr.edu.iyte.caffeine/u0a397 for service tr.edu.iyte.caffeine/.CaffeineTileService
12-04 19:10:37.006  6716  6716 E SchedPolicy: open of /dev/cpuctl/bg_non_interactive/tasks failed: No such file or directory
12-04 19:10:37.119  6716  6716 I CaffeineTileService: started listening
12-04 19:10:37.119  6716  6716 I CaffeineTileService: phone is locked, caffeine won't operate
12-04 19:10:37.120  6716  6716 I CaffeineTileService: updating label: Caffeine
12-04 19:17:10.399  6716  6716 D AndroidRuntime: Shutting down VM
12-04 19:17:10.399  6716  6716 E AndroidRuntime: FATAL EXCEPTION: main
12-04 19:17:10.399  6716  6716 E AndroidRuntime: Process: tr.edu.iyte.caffeine, PID: 6716
12-04 19:17:10.399  6716  6716 E AndroidRuntime: java.lang.IllegalArgumentException: Service not registered: tr.edu.iyte.caffeine.CaffeineTileService$e@862a4ea
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.app.LoadedApk.forgetServiceDispatcher(LoadedApk.java:1475)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.app.ContextImpl.unbindService(ContextImpl.java:1637)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.content.ContextWrapper.unbindService(ContextWrapper.java:697)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at tr.edu.iyte.caffeine.CaffeineTileService.onStopListening(Unknown Source:19)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.service.quicksettings.TileService$H.handleMessage(TileService.java:401)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:105)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6694)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
12-04 19:17:10.399  6716  6716 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:769)
12-04 19:17:10.416  1719  1900 I ActivityManager: Showing crash dialog for package tr.edu.iyte.caffeine u0
12-04 19:17:11.681  1719  2862 I ActivityManager: Killing 6716:tr.edu.iyte.caffeine/u0a397 (adj 0): crash
```

**Comment by Efreak** [comment]
The file may not exist at all, as the following commands show from termux: 
```
$ stat /dev/cpuctl /dev/cpuctl/bg_non_interactive /dev/cpuctl/non_interactive/tasks
  File: /dev/cpuctl
  Size: 0         	Blocks: 0          IO Block: 4096   directory
Device: 12h/18d	Inode: 1           Links: 2
Access: (0555/dr-xr-xr-x)  Uid: ( 1000/  system)   Gid: ( 1000/  system)
Access: 1970-02-02 11:02:03.670000714 -0800
Modify: 1970-02-02 11:02:03.670000714 -0800
Change: 1970-02-02 11:02:03.670000714 -0800
 Birth: -
stat: cannot stat '/dev/cpuctl/bg_non_interactive': No such file or directory
stat: cannot stat '/dev/cpuctl/non_interactive/tasks': No such file or directory
```

**Comment by emrsmsrli** [comment]
Hey there! I'm not maintaining FDroid version, since I'm not the one who uploaded the app there. You should download Google Play version instead. It's more up to date. But I can't guarantee that even GP version will work because I stopped maintaining the app entirely. Thanks.

**Comment by Efreak** [comment]
Thanks. I think the update fixed it. I installed from F-Droid because that's where I saw it; I assumed it was an official F-Droid release, but it seems to be from a 3rd-party repo that doesn't do its own builds, as Google play allowed a direct update (ie no signing issues).

Anyways, thanks much for the app! This and dynamic rotation lock (overlay icon) seem like such obvious functions, I can't understand why they're not built into Android by now.
